starter-code/js/index.js

```diff
@@ -1,52 +1,76 @@
-function deleteItem(e){
+// getTotalPrice for 1st iteration
+// just a single product
 
-}
+// function getTotalPrice() {
+//   var price = document.querySelectorAll(".price")[0].innerHTML;
+//   var quantity = document.querySelectorAll(".quantity")[0].value;
+//   var total = price * quantity;
+//   document.querySelectorAll(".subTotal-price")[0].innerHTML = "$ " + total;
+// }
 
-function getPriceByProduct(itemNode){
+// getTotalPrice for 2nd iteration
+// 2 products
 
-}
+function getTotalPrice() {
+  
+  var sumPrice = 0;
+    for (var i=0; i< document.querySelectorAll(".product").length; i++) {
+      var price = document.querySelectorAll(".price")[i].innerHTML;
+      var quantity = document.querySelectorAll(".quantity")[i].value;
+      var total = price[i] * quantity[i];
+      sumPrice += total
+      document.querySelectorAll(".subTotal-price")[i].innerHTML = "$ " + total;
+    }
+ }  
```

Comment:
I recommend you this approach for this function since only we do the `querySelectorAll()` once:
```
function getTotalPrice() {

  var sumPrice = 0;
  var products = document.querySelectorAll(".product");
  var prices = document.querySelectorAll(".price");
  var quantities = document.querySelectorAll(".quantity");
  var subTotal = document.querySelectorAll(".subTotal-price");
  
  for (var i = 0; i < products.length; i++) {
    var price = prices[i].innerHTML;
    var quantity = quantities[i].value;
    var total = Number(price) * Number(quantity);
    subTotal[i].innerHTML = "$ " + total;
  }
}
```